elasticsearch/linux_distributions/build.gradle

```diff
@@ -66,38 +66,58 @@ ext {
     new ByteArrayOutputStream().withStream { os4 ->
       def result4 = exec {
         executable = '../../bin/plugins-info'
-        args = ['rpm', '--git']
+        args = ['elasticsearch', 'git', '--require-install-true']
         standardOutput = os4
       }
       ext.plugin_git = os4.toString().replaceAll("\n", ",")
     }
 
     new ByteArrayOutputStream().withStream { os5 ->
       def result5 = exec {
-        executable = '../../.github/scripts/plugin_tag.sh'
-        args = [ext.plugin_git, opendistroVersion]
+        executable = '../../bin/version-info'
+        args = ['--is-cut']
         standardOutput = os5
       }
-      ext.plugin_versions = os5.toString().replaceAll("v", "")
+      ext.is_cut = os5.toString().trim()
+    }
+    println("is_cut: " + ext.is_cut)
+
+   if (ext.is_cut.equals("true")) {
+      new ByteArrayOutputStream().withStream { os6 ->
+        def result6 = exec {
+          executable = '../../bin/plugins-info'
+          args = ['elasticsearch', 'cutversion', '--require-install-true']
+          standardOutput = os6
+        }
+        ext.plugin_versions = os6.toString()
+        println("Version is cut")
+      }
+    }
+    else {
+      new ByteArrayOutputStream().withStream { os7 ->
+        def result7 = exec {
+          executable = '../../.github/scripts/plugin_tag.sh'
+          args = [ext.plugin_git, opendistroVersion]
+          standardOutput = os7
+        }
+        ext.plugin_versions = os7.toString().replaceAll("v", "")
+        println("Version not cut")
+      }
     }
 
-    new ByteArrayOutputStream().withStream { os6 ->
-      def result6 = exec {
+    new ByteArrayOutputStream().withStream { os8 ->
+      def result8 = exec {
         executable = '../../bin/version-info'
         args = ['--od-next']
-        standardOutput = os6
+        standardOutput = os8
       }
-      ext.plugin_versions_next = os6.toString().trim() + ".0"
+      ext.plugin_versions_next = os8.toString().trim() + ".0"
```

Comment:
Do you need to rename these variables? Surely they're only in scope inside the closure.

Reply:
I do not have to, I just like to make sure I know which step is going first, especially they seems very similar.
And it is easier to check errors when you know the only os1 is having issues, not the second / third os1.